nova/virt/vmwareapi/vm_util.py

```diff
@@ -931,8 +932,9 @@ def get_vmdk_info(session, vm_ref, uuid=None):
         if device.__class__.__name__ == "VirtualDisk":
             if device.backing.__class__.__name__ == \
                     "VirtualDiskFlatVer2BackingInfo":
-                if not root_device or root_device.controllerKey > device.controllerKey or \
-                                        root_device.controllerKey == device.controllerKey \
+                if not root_device \
+                    or root_device.controllerKey > device.controllerKey \
+                    or root_device.controllerKey == device.controllerKey \
                         and root_device.unitNumber > device.unitNumber:
```

Comment:
Is this formatted automatically? Never seen it like that ... multiple lines of 1 indent just to have a 2 indent at the end and then continue with the 1 indent body ... quite confusing to read.

Reply:
That was done manually: I put each `or`-clause on its own line, and indented `and` by an additional line to make clear that it is a sub-clause to the `or`-clause before.